docs/endpoints/trigger_run_custom.md

```diff
@@ -0,0 +1,3 @@
+::: prefect_hightouch.api_client.api.default.trigger_run_custom
+    options:
+      filters: ["!^__[^__]", "!Response"]
```

Comment:
is there a way we could set this filter in the mkdocs.yaml if its consistent among all these default endpoints?